.github/workflows/make-ci.yaml

```diff
@@ -0,0 +1,38 @@
+---
+name: Make CI
+
+'on':
+  workflow_call:
+    inputs:
```

Comment:
I would add several more inputs here as there were in `python-lint.yaml`. They can all have sane defaults, but there are sometimes reasons you would need to change the Python version, change the working directory, etc.

Reply:
discussed on meet, use tox or something if you want to test over multiple python versions. As far as changing directories, there's no real reason to put the makefile anywhere other than top level